include/picongpu/particles/ionization/byField/ADK/ADK_Impl.hpp

```diff
@@ -147,6 +152,9 @@ namespace ionization
                 const T_WorkerCfg & workerCfg
             )
             {
+                /* shift origin of jbox to cell of particle */
+                jBox = jBox.shift(blockCell);
```

Comment:
The comment seems to not match the code. Here the box is shifted to start of supercell (block), not cell.